_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,392 @@
+`Ctrl-Shift` combinations both of which weren't previously possible in the
+terminal. Be sure to [read more][tui-bindings] about the updated binding
+capabilities for additional examples and possibilities.
+
+### Builtin Terminal Emulation
+
+@tarruda has finished [his pull request][term-pr] adding a new `:terminal`
+command which when ran launches a full terminal emulator. The terminal is using
+Neovim's native buffers and windows which means the user can enter [normal
+mode][term-normal] by pressing `<C-\><C-n>`.
+
+This is meant to replace and provide more capabilities than the already existing
+Vim command, `:shell`, which just launches a shell. More explanation on the
+decision of this feature can be found in [this comment][term-comment] made by
+@tarruda.
+
+The complete documentation for this feature can be found under [`:h
+nvim-terminal-emulator`][term-docs].
+
+This feature is implemented using another of @leonerd's projects called
+[libvterm][term-libvterm].
+
+### True Colors
+
+With the changes to the TUI described above, this allowed @tarruda to
+make a [pull request][color-pr] to add true colors to terminals that support it.
+
+A more detailed explanation of what true colors means as well as terminals that
+support it can be found in [this Gist][color-info] that was written by @XVilka.
+
+To enable true colors, set the environment variable
+`NVIM_TUI_ENABLE_TRUE_COLOR`.
+
+### Special Interfaces Cleaning
+
+Vim has a number of ways to start it through its many aliases. The complete list
+can be found under the [starting section][in-starting] on the Vimdocs.
+
+@Pyrohh opened [an issue][in-discussion] to discuss the removal of such
+aliases. All of the aliases had corresponding flags that could be used instead.
+
+After some discussion it was decided to remove the lesser used ones. @Pyrohh
+then made a [pull request][in-pr] to remove the following:
+
+- [`ex`][in-ex]: Ex mode.
+- [`exim`][in-exim]: "Improved" Ex mode.
+- [`view`][in-view]: Read-only mode.
+- [`gvim`][in-gvim]: Using the GUI.
+- [`gex`][in-gex]: GUI in Ex mode.
+- [`gview`][in-gview]: GUI in read-only mode.
+- [`rvim`][in-rvim]: Restricted mode.
+- [`rview`][in-rview]: Read-only and in restricted mode.
+- [`rgvim`][in-rgvim]: GUI and in restricted mode.
+- [`rgview`][in-rgview]: GUI, read-only, and in restricted mode.
+
+#### Removal of "Easy Mode"
+
+@fmoralesc also opened a [pull request][easy-pr] to remove `evim`. Which given
+by [the docs][easy-vimdoc] says it starts in insert mode which allows it to act
+like a normal editor.
+
+Even with removing of the feature, emulating it is still possible by using the
+underlying
+[`'insertmode'`](http://neovim.org/doc/user/options.html#'insertmode') option
+that easy mode leveraged.
+
+#### Merging of vimdiff and -d
+
+In a separate [pull request][vimdiff-pr], @Pyrohh, merged
+[`vimdiff`][vimdiff-docs] and `vim -d` which were two different ways of viewing
+a diff in Vim.
+
```

Comment:
> I'm going to keep the easy mode section since you didn't make that PR.

Well there's that, but the reason I didn't comment on easy mode is because it's more than just a symlink and some docs (runtime file, some code IIRC).